build.sbt

```diff
@@ -54,7 +55,7 @@ lazy val commonSettings = Seq(
   publishMavenStyle := true,
   publishArtifact in Test := false,
   pomIncludeRepository := { _ => false },
-  javaVersionPrefix in javaVersionCheck := Some("1.7")
+  javaVersionPrefix in javaVersionCheck := Some("1.8")
```

Comment:
until we move to spark 1.6 we'll have to stick with java 7 unfortunately: https://issues.apache.org/jira/browse/SPARK-6152